foundations/html_css/css-foundations/intro-to-css.md

```diff
@@ -80,6 +80,8 @@ Class selectors will select all elements with the given class, which is just an
 ```
 
 Note the syntax for class selectors: a period immediately followed by the case-sensitive value of the class attribute. Classes aren't required to be specific to a particular element, so you can use the same class on as many elements as you want.
+>The selector will not work if the class name begins with a digit, e.g. `.4lert-text` will not select an element even if it has the class `4lert-text` - for now, just avoid leading digits in class selector names.
```

Comment:
Nit: I think this can be reworded a little to make it clearer:

```suggestion
Class selectors won’t work if the class name begins with a number. For example, if you give an element the class name `.4lert-text`, using `.4lert-text` as a selector won’t match it.
```